app.js

```diff
@@ -1,28 +1,26 @@
-const express = require('express');
+const express = require("express");
 const app = express();
-const { getCategories } = require ('./controller')
-
-app.get('/api/categories', getCategories)
-
+const { getCategories } = require("./controller");
 
+app.get("/api/categories", getCategories);
+app.get("/api/reviews", getReviews)
 
 app.use((err, req, res, next) => {
-    if (err.status) {
-      res.status(err.status).send(err);
-    } else next(err);
-  });
+  if (err.status) {

```

Comment:
have a think about how to check for a fairly generic message to give your user some feedback too